app/assets/javascripts/surveys.js

```diff
@@ -148,7 +81,27 @@
       return activeSurvey;
     },
 
+    getDefaultSurvey: function() {
+      try {
+        return JSON.parse($('#user-satisfaction-survey-container [data-survey-default]').text());
```

Comment:
Is `JSON.parse` safe to use across the browsers we expect to serve?  [Can I Use](http://caniuse.com/#feat=json) suggests it's widely supported, but I'm not sure what our browser support matrix is.